### PR TITLE
Feature/add resources into side containers

### PR DESCRIPTION
**Pull request by informalict**
DON'T MERGE!!!

IBM CV requires that created pods need to have the resource requirements.
Integration test TestAddingMetrics does not work in version 3.5.2 of arangod

### CODE REVIEW

File: pkg/util/k8sutil/pods.go [hunk 1]
```diff
@@ -536,19 +548,20 @@ type ArangodbExporterContainerConf struct {
...
 // If the pod already exists, nil is returned.
 // If another error occurs, that error is returned.
 func CreateArangodPod(kubecli kubernetes.Interface, developmentMode bool, deployment APIObject,
-	role, id, podName, pvcName, image, lifecycleImage, alpineImage string,
-	imagePullPolicy v1.PullPolicy, imagePullSecrets []string,
-	engine string, requireUUID bool, terminationGracePeriod time.Duration,
-	args []string, env map[string]EnvValue, finalizers []string,
-	livenessProbe *HTTPProbeConfig, readinessProbe *HTTPProbeConfig, tolerations []v1.Toleration, serviceAccountName string,
-	tlsKeyfileSecretName, rocksdbEncryptionSecretName string, clusterJWTSecretName string, nodeSelector map[string]string,
-	podPriorityClassName string, resources v1.ResourceRequirements, exporter *ArangodbExporterContainerConf, sidecars []v1.Container, vct *v1.PersistentVolumeClaim) error {
```
Comment (maierlars) [inline pkg/util/k8sutil/pods.go]: I am waiting for the day that someone cleans this up.

Comment (informalict) [reply pkg/util/k8sutil/pods.go]: https://github.com/arangodb/kube-arangodb/pull/495#issue-336725977

Comment (informalict) [reply pkg/util/k8sutil/pods.go]: Test created. 
Pods are rotated. 
I have created a new task for cleaning up.

**Comment by informalict** [comment]
It can not be merged because of https://github.com/arangodb/kube-arangodb/pull/501. There will be a lot of conflicts and it is easier and faster to rewrite that task once again

**Pull request closed without merging**